src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -431,7 +431,7 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			fluid.dripParticle = new Particle(modElement.getWorkspace(),
 					getRandomDataListEntry(random, ElementUtil.loadAllParticles(modElement.getWorkspace())));
 			fluid.tintType = getRandomString(random,
-					Arrays.asList("No tint", "Grass", "Foliage", "Water", "Sky", "Fog", "Water fog"));
+					Arrays.asList("No tint", "Grass", "Foliage", "Foliage birch", "Foliage spruce", "Default foliage", "Water", "Sky", "Fog", "Water fog"));

```

Comment:
This is the tint type list of fluids, not blocks

Reply:
I am no words but woops